docs/cis_self_assessment.md

```diff
@@ -534,46 +452,20 @@ Kubernetes makes use of a number of key files as part of the operation of its co
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
-
-``` bash
-chmod -R 600 /etc/kubernetes/ssl/certs/serverca
-```
-
-**Audit Script:** 1.1.21.sh
-<!-- TODO escaped asterisk -->
-```
-#!/bin/bash -e
-check_dir=${1:-/etc/kubernetes/ssl}
+**Audit**
+Run the below command on the master node.
 
-for file in $(find ${check_dir} -name "\*key.pem"); do
-file_permission=$(stat -c %a ${file})
-if [[ "${file_permission}" == "600" ]]; then
-continue
-else
-echo "FAIL: ${file} ${file_permission}"
-exit 1
-fi
-done
+For example, 
 
-echo "pass"
-```
-
-**Audit Execution:**
-
-```
-./1.1.21.sh /etc/kubernetes/ssl
+```bash
+stat -c %n\ %a /var/lib/rancher/rke2/server/tls/*.key
 ```
 
-**Expected result**:
-
-```
-'pass' is present
+**Remediation:**
```

Comment:
This remediation is still the `chmod` command. Needs to be our explanation instead